### DISCUSSION TITLE
Arabic nlp question

### DISCUSSION THREAD

**Discussion by IsaacMQ** [community/community#56519] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

I want to do a question about arabic nlp but i've not discovered how.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @IsaacMQ , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Reply by IsaacMQ** [reply]
Only i want to ask about if in NLP there's a way to find the different nuances of arabic synonyms.

**Reply by IsaacMQ** [reply]
Please,someone can help me?

**Comment by IsaacMQ** [comment]
Is there's a way in NLP to find the different nuances of arabic synonyms?
________________________________
De: Lili ***@***.***>
Enviado: miércoles, 31 de mayo de 2023 10:44 a. m.
Para: community/community ***@***.***>
Cc: IsaacMQ ***@***.***>; Mention ***@***.***>
Asunto: Re: [community/community] Arabic nlp question (Discussion #56519)


Hi @IsaacMQ<https://github.com/IsaacMQ> , we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by IsaacMQ** [comment]
ANSWER ME,please.
